docs/examples/color_space_conversion.ipynb

```diff
@@ -0,0 +1,169 @@
+{
```

Comment:
Suggestion/nitpicking: Adding some markdown cell explaining each code cell would make (something really simple like **Defining the pipeline**) would make this notebook easier to navigate through